.github/workflows/testing_and_building_repos.yml

```
@@ -3,18 +3,21 @@ on: [push, pull_request]
 jobs:
   build: 
     strategy:
+      fail-fast: false
       matrix:
         os: ['ubuntu-18.04']
-        perl: ['5.26'] 
-        
+        perl: ['5.26']
+        node: ['14.8.0']
+
     runs-on: ${{matrix.os}} 
-    
+
     defaults:
       run:
         shell: bash -l -e -o pipefail {0}
 
     env:
       WSI_CONDA_CHANNEL: "https://dnap.cog.sanger.ac.uk/npg/conda/devel/generic"
+      NPM_CONFIG_PREFIX: "$HOME/.npm-global"
 
     name: Perl ${{ matrix.perl }} on ${{ matrix.os }}  
```

(v. minor)
```suggestion
    name: Perl ${{ matrix.perl }} and Node ${{ matrix.node }} on ${{ matrix.os }}  
```
maybe?